File: 240201/시험 통과 여부 확인하기 2/verify-test-passed-2.py
n = int(input())
count = 0
for _ in range(n):
    arr = list(map(int, input().split()))
    new_arr = []
    for i in arr:
        new_arr.append(i)

        sum_arr = sum(new_arr)
        div_arr = sum_arr // 4

    # print(div_arr)

    if div_arr >= 60:
        answer = 'pass'
    else:
        answer = 'fail'

    print(answer)
    
    if answer == 'pass':
        count += 1
print(count)

File: 240921/시험 통과 여부 확인하기 2/verify-test-passed-2.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int n ;
    cin >> n;

    int total = 0;
    for (int i = 1; i <= n; i ++) {
        int arr[4];
        int sum = 0;
        int cnt = 0;
        for (int i = 0; i < 4; i ++) {
            cin >> arr[i];

            sum += arr[i];
            cnt ++;


        }
        if ((sum / cnt) >= 60) {
            cout << "pass" << endl ;
            total ++;
        }
        else {
            cout << "fail" << endl;
        }
    }
    cout << total;

    return 0;
}